server/infra/database/CountryRepository.ts

```diff
@@ -59,102 +59,155 @@ export default class CountryRepository extends BaseRepository<Country> {
     return object.rows;
   }
 
-  // eslint-disable-next-line class-methods-use-this
-  getLeaderBoard(/* top = 10 */) {
-    // const sql = `
-    //   select r.*, region.name, ST_AsGeoJSON(centroid) as centroid  from (
-    //   select count(region.id) as planted, region.id
-    //   from trees
-    //   LEFT JOIN region
-    //   on ST_WITHIN(trees.estimated_geometric_location, region.geom)
-    //   left join region_type
-    //   on region.type_id = region_type.id
-    //   where
-    //   region_type.type = 'country'
-    //   group by region.id
-    //   order by count(region.id) desc
-    //   limit ${top}
-    //   ) r left join region
-    //   on r.id = region.id
-    //   ;
-    // `;
-    // const object = await this.session
-    //   .getDB()
-    //   .raw(sql);
-    // return object.rows;
+   
+  async getLeaderBoard(region: string /* top = 10 */) {
+    let regionID = 0;
+
+    switch (region.toUpperCase()) {
+      case 'AFRICA':
+        regionID = 6632611;
+        break;
+      case 'ANTARCTICA':
+        regionID = 6632612;
+        break;
```

Comment:
@dagmawig we can not use fixed id here, because they could be different in envs, the id is randomly generated. 

You should join the table and use the name (Africa) to query the data